client/src/pages/ChefProfile.js

```diff
@@ -1,15 +1,94 @@
-/**Comment added to avoid merge conflict with ongoing development  */
-// import React, { useContext } from "react";
-// import { AuthContext } from "../context/auth-context";
+import React, { useState, useEffect, useContext } from "react";
+import { Grid, Hidden, Box, Typography, Paper } from "@material-ui/core";
+import ChefSideBar from "../components/ChefProfile/ChefSideBar";
+import ChefRecipes from "../components/ChefProfile/ChefRecipes";
+import { makeStyles } from "@material-ui/core/styles";
+import { RecipeContext, RecipeDispatchContext } from "../context/RecipeContext";
+import { UserContext } from "../context/UserContext";
+import axios from "axios";
+import { getRecipesByChef } from "../actions/recipeActions";
+
+const ChefProfile = ({ history }) => {
+  const useStyles = makeStyles((theme) => ({
+    chefMenuName: {
+      marginTop: theme.spacing(10),
+      marginBottom: theme.spacing(6),
+    },
+  }));
+
+  const classes = useStyles();
+
+  const { recipes } = useContext(RecipeContext);
+  const dispatch = useContext(RecipeDispatchContext);
```

Comment:
Yes we have discussed this and once her branch is merged with the dev I can get this updated on my branch

Reply:
The PR is huge because we merged the Demo branch with Dev yesterday. Since the dev is up to date now, I will create a new PR that will only have my checkout page. Sorry for the confusion, I will close this PR.